blockchain/state/iterator.go

```diff
@@ -37,6 +37,18 @@ var (
 	errStopByQuit = errors.New("stopByQuit")
 )
 
+type NodeParallelIterator struct {
+	iterators []*NodeIterator
+}
+
+// NewNodeIterator creates an post-order state node iterator.
+func NewNodeParallelIterator(state *StateDB) *NodeIterator {
```

Comment:
`NodeParallelIterator` and `NewNodeParallelIterator ` are used nowhere.

Reply:
I removed this unnecessary code. :)